sources/packages/backend/apps/api/src/services/education-program-offering/education-program-offering-validation.models.ts

```diff
@@ -363,6 +370,13 @@ export class OfferingValidationModel {
   /**
    * Program related costs.
    */
+  @ValidateIf(
+    (offering: OfferingValidationModel) =>
+      !!offering.actualTuitionCosts &&
+      !!offering.programRelatedCosts &&
+      !!offering.mandatoryFees &&
+      !!offering.exceptionalExpenses,
```

Comment:
A common function can be created to centralize the logic. Please see below, if it does not work something similar can probably be used.
```ts
function canValidateCosts(offering: OfferingValidationModel): boolean {
  return (
    !!offering.actualTuitionCosts &&
    !!offering.programRelatedCosts &&
    !!offering.mandatoryFees &&
    !!offering.exceptionalExpenses
  );
}
```

```ts
@ValidateIf(canValidateCosts)
```